app/src/main/res/layout/fragment_timeline_notifications.xml

```diff
@@ -61,12 +61,6 @@
 
     </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
-    <ProgressBar
-        android:id="@+id/progressBar"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_gravity="center" />
-
```

Comment:
Why's that.

To me it seems weird that there are two different styles of progress animation for the user, even though they're conveying the same information.

This is something I want to clean up when I modify the other fragments.

Reply:
For me the swipe refresh spinner is exclusively for indicating that the user triggered refresh is in progress. Everytime I see it when I didn't pull it down it feels weird and i wonder if the app is working correctly. The progress bar on the other hand reassures me that there really is nothing here and the app still loads stuff. It might be my personal preference but as far as I can tell this is also like this in other apps.

Reply:
This is the only thing preventing me from merging this

Reply:
That's annoying, GH has been hiding this behind a "hidden conversations" message for weeks.

I'll look at this, but I still think it's unnecessary complication in the code and the layout.